src/main/java/tfifteenfour/clipboard/logic/parser/NewEditCommandParser.java

```diff
@@ -50,8 +49,8 @@ public EditCommand parse(String args) throws ParseException {
         try {
             addCommandType = CommandTargetType.fromString(ArgumentTokenizer.tokenizeString(args)[1]);
```

Comment:
ohno this variable name should be `editCommandType`, my mistake HAHA

Reply:
Ok!